### PR TITLE
docs: document our Node.js versioning policy

### PR DESCRIPTION
**Pull request by codebytere**
#### Description of Change

As in title. Follow-up from summit.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation, tutorials, templates and examples are changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/main/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/main/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

**Review by VerteDinde — APPROVED** [review]

**Review by dsanders11 — APPROVED** [review]

File: docs/tutorial/electron-timelines.md [hunk 1]
```diff
@@ -79,6 +73,36 @@ and the version prior to that receives the vast majority of those fixes
...
+### Chromium Version Support
+
+:::info Chromium release schedule
+
+Chromium's public release schedule is [here](https://chromiumdash.appspot.com/schedule).
+
+:::
+
+Electron targets Chromium even-number versions, releasing every 8 weeks in concert
+with Chromium's 4-week release schedule.
```
Comment (erickzhao) [inline docs/tutorial/electron-timelines.md:85]:
```suggestion
Electron targets Chromium even-number versions, releasing every 8 weeks in concert
with Chromium's 4-week release schedule. For example, Electron 26 uses Chromium 116, while Electron 27 uses Chromium 118.
```

File: docs/tutorial/electron-timelines.md [hunk 2]
```diff
@@ -79,6 +73,36 @@ and the version prior to that receives the vast majority of those fixes
...
+
+### Node.js Version Support
+
+Electron upgrades Node.js to even-number versions when they enter Active LTS. The schedule
```
Comment (erickzhao) [inline docs/tutorial/electron-timelines.md:89]: I think there's a phrasing improvement to be made about when the Node.js LTS version upgrade happens.

When a new Chromium stable drops, an Electron stable version with the corresponding Chromium version will also be available. With Node.js, only the `main` branch gets updated at that point in time, right?

Comment (codebytere) [reply docs/tutorial/electron-timelines.md:89]: Edited it - i think it should be a bit clearer now!

**Review by erickzhao — APPROVED** [review]
docs: lgtm

**Pull request merged**